packages/core/src/external/fhir/consolidated/deduplicate.ts

```diff
@@ -12,19 +13,23 @@ export function deduplicate({
   patientId: string;
   bundle: Bundle<Resource>;
 }): Bundle<Resource> {
+  const { log } = out(`Hydrating FHIR for cxId ${cxId}, patientId ${patientId}`);
```

Comment:
"deduplication", not "hydrating", with?

Reply:
nit: ...and, if we can make this string smaller, the better so it helps reading the logs (like, replace ` for ` by `, can likely remote `FHIR` as hydration is always on FHIR, `cxId` can become `cx`, `patientId` can become `pt`